packages/keyring-api/src/index.ts

```diff
@@ -1,13 +1,8 @@
 export * from './api';
 export * from './btc';
-export type * from './contexts';
-export * from './eth';
 export * from './sol';
+export * from './eth';
+export type * from './contexts';
+export * from './rpc';
 export * from './events';
-export * from './internal';
-export * from './JsonRpcRequest';
-export * from './KeyringClient';
-export * from './KeyringSnapRpcClient';
-export * from './rpc-handler';
-export * from './snap-utils';
-export * from './superstruct';
+export * from './utils';
```

Comment:
Only contains the new `Paginate` types, IDK we should keep those in a `utils` folder?

Reply:
This sub-folder has been removed now (merged with the `api` folder for the moment).